math-latex.mim

```diff
@@ -131,7 +131,7 @@ you don't have to turn off this input method to input normal ASCII characters.")
     ("succ"       "≻")
     ("succeq"       "≽")
 
-    ("in"          "∈")
+    ("In"          "∈")
```

Comment:
Given that ∈ is used far more than 𝑛 - might it be better to use In for the latter?

Reply:
Yes, that does make sense. As long as ∈ and 𝑛 have distinct keystroke sequences, the problem is solved.